rest/route/agent.go

```diff
@@ -1527,7 +1526,6 @@ func (h *checkRunHandler) Run(ctx context.Context) gimlet.Responder {
 		errorMessage := fmt.Sprintf("created checkRun not return for task: '%s'", t.Id)
 		grip.Error(message.Fields{
 			"message": errorMessage,
-			"error":   err.Error(),
```

Comment:
This line won't get reached if the error is not nil (my linter was yelling at me this will panic bc err is nil bc of the check on 1519 `if err != nil {`